### PR TITLE
fix(tooltip): prevent tooltip to create horizontal scrollbars

### PR DESCRIPTION
**Pull request by multiwebinc**
## Description

When the window is resized, occasionally the tooltip creates horizontal or vertical scrolling. This PR sets the `left` and `top` attributes to `0` when the tooltip is closed.

As an alternative, we could set `display: none` when hidden instead of `visibility: hidden`.

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Tested locally by use

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/lib/components/Floating/Floating.tsx [hunk 1]
```diff
@@ -125,8 +125,8 @@ export const Floating: FC<FloatingProps> = ({
           ref: floating,
           style: {
             position: strategy,
-            top: y ?? ' ',
-            left: x ?? ' ',
+            top: open ? y ?? ' ' : 0,
```
Comment (tulup-conner) [inline src/lib/components/Floating/Floating.tsx:128]: We can just change these to open ? y : 0 now, right?

**Review by tulup-conner — CHANGES_REQUESTED** [review]

**Comment by rluders** [comment]
Can we drop this one in favor of #683? @multiwebinc @tulup-conner

**Comment by multiwebinc** [comment]
@rluders I'm off for the weekend. I'll take a look at it next week unless someone else wants to. But I think they are probably complimantary because I didn't alter the styles much in #683.

**Comment by rluders** [comment]
> @rluders I'm off for the weekend. I'll take a look at it next week unless someone else wants to. But I think they are probably complimantary because I didn't alter the styles much in #683.

Not being pushy on this one, but... any update?

**Comment by multiwebinc** [comment]
@rluders Sorry for the delay. Yes, I see that this can just be closed now in favor of #683.

**Pull request closed without merging**